func/zenity.py

```diff
@@ -0,0 +1,12 @@
+
+import os,sys
+
+#Check if Zenity is installed
+checkzenity = os.system('zenity --version')
+
+def zenity_popup(title, text, type="info", width=300):
+    if checkzenity == 0 and "deck" not in os.path.expanduser("~"):
```

Comment:
I feel you should bring back the `if "deck" not in os.path.expanduser("~"):` in steam.py and just do `checkzenity == 0` here. 
Helps keep it simple and avoid confusion.

Reply:
Will do!